lib/Less/Functions.php

```diff
@@ -813,11 +813,11 @@ private function _isa($n, $type) {
 		return is_a($n, $type) ? new Less_Tree_Keyword('true') : new Less_Tree_Keyword('false');
 	}
 
-	public function tint($color, $amount) {
+	public function tint($color, $amount = null) {
```

Comment:
@PhrozenByte You're absolutely right. I shouldn't have made that assumption. I will fix this.